org.mixedrealitytoolkit.uxcore/Button/PressableButton.cs

```diff
@@ -640,38 +640,36 @@ private float IntersectQuad(Vector3 start, Vector3 end, float depth)
         }
 
         /// <summary>
-        /// Registers the duple Collider + XRBaseInteractor as triggering proximity.
+        /// Registers the interactable triggering proximity.
         /// </summary>
-        /// <param name="collider">Collider triggering proximity.</param>
-        /// <param name="xrBaseInteractor">Interactor triggering proximity.</param>
-        public void OnProximityEntered(ProximityEnteredEventArgs proximityEnteredEventArgs)
+        /// <param name="xrBaseInteractable">Interactable triggering proximity.</param>
+        public void OnProximityEntered(ProximityHoverEnteredEventArgs proximityEnteredEventArgs)
         {
-            if (proximityEnteredEventArgs.collider != null &&
-                activeCollidersWithInteractor.Add((proximityEnteredEventArgs.collider, proximityEnteredEventArgs.interactor)) &&
-                activeCollidersWithInteractor.Count >= 1)
+            if (activeInteractables.Add(proximityEnteredEventArgs.Interactable as XRBaseInteractable))
```

Comment:
The interactable here will always be the button itself.   The args should be the interactor (or the ProximityDetector) that is proximity hovering over the button.

Why was this change made?

Reply:
I got confused from a previous feedback and was under the wrong impression that and Interactable would be enough to uniquely identify the source of the event but I was wrong.  I reverted to using Interactors in https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/659/commits/21f16fc4597f04b28bbe910fcfd1c1bc6a21adf0
Thank you for the offline explanation on this issue by the way.